client/components/backup-retention-management/retention-options/retention-option-card.tsx

```diff
@@ -52,7 +52,7 @@ const RetentionOptionCard: FunctionComponent< RetentionOptionCardProps > = ( {
 			</div>
 			<div className="retention-option__space-needed">
 				<div className="space-needed__label">{ translate( 'Space needed:' ) }</div>
-				<div className="space-needed__value">{ spaceNeeded }</div>
+				<div className="space-needed__value">{ useStorageText( spaceNeededInBytes ) }</div>
```

Comment:
You might want to move the hook call to the top of the function. Not a requirement, but it gives better visibility on the fact that we're calling a hook and reduces the chance of someone breaking the [rules of hooks](https://reactjs.org/docs/hooks-rules.html) in the future.

Reply:
Done